### ISSUE TITLE
Entire training code

### ISSUE THREAD

**Issue by rajeswar18** [shjo-april/RecurSeed_and_EdgePredictMix#2]
Hi,
Great work so far, 
we are currently building on this work. It would be great if you can release the whole code rather than a template.
Looking forward to the code.

**Comment by RShevcheCape** [comment]
Same here :) Any updates on this matter, @OFRIN ?

**Comment by shjo-april** [comment]
Hi, thanks for your interest in our paper!

The paper is under review. We will release our code after the blind review, which may need a few months. We also improved the 2~3% mIoU score by replacing the proposed CM with the new data augmentation method and adding another loss in the proposed RS.

[~2022/08/15] Update the paper in Arxiv and the title in Github.
[~2022/11/30] Release the entire training and evaluation codes.

**Comment by tanbin5070** [comment]
Hi,wait for the author to release the source code. Thank you very much
